tests/firedrake/vertexonly/test_vertex_only_mesh_generation.py

```diff
@@ -434,7 +434,7 @@ def test_on_boundary_behaviour():
     coords = np.array([[0.4, 0.2, 0.3]])
     mesh = UnitCubeMesh(10, 10, 10)
     vm = VertexOnlyMesh(mesh, coords)
-    total_num_cells = MPI.COMM_WORLD.allreduce(len(vm.coordinates.dat.data_ro_with_halos), op=MPI.SUM)
+    total_num_cells = MPI.COMM_WORLD.allreduce(len(vm.coordinates.dat.data_ro), op=MPI.SUM)
```

Comment:
why was this necessary?

Reply:
I think this is probably the right thing to do. The total number of cells shouldn't rely on anything to do with halos.

Reply:
I might actually advocate removing this test. It is extremely unclear what it is actually testing.

Reply:
I'd be hesitant to remove it without knowing what it was added for originally. We could bring it to the meeting tomorrow.